slack-api-model/src/main/java/com/slack/api/model/block/element/RichTextSectionElement.java

```diff
@@ -54,7 +54,7 @@ public static class Link implements RichTextElement {
     public static class Date implements RichTextElement {
         public static final String TYPE = "date";
         private final String type = TYPE;
-        private String timestamp;
+        private Long timestamp;
```

Comment:
Nice catch 💯 I think this may be an `integer` on the backend, but this project uses `Long` for other timestamp values